coinswap-swift/build-xcframework-ci.sh

```
@@ -0,0 +1,47 @@
+
+cd ../coinswap-swift/ || exit
+
+rm -rf "./coinswap_ffi.xcframework"
+
+xcodebuild -create-xcframework \
+    -library "${TARGETDIR}/${MAC_TARGET}/${PROFILE_DIR}/libcoinswap_ffi.a" \
+    -headers "${SWIFT_OUT_DIR}/${HEADER_BASENAME}" \
+    -output "./coinswap_ffi.xcframework"
+
+# Keep Swift sources clean: only .swift files should stay in the package Sources dir
+rm -rf "${SWIFT_OUT_DIR}/${HEADER_BASENAME}"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)